omero/sysadmins/server-syslog.txt

```diff
@@ -0,0 +1,134 @@
+    using `SIFT`.
+
+
+Configuration
+^^^^^^^^^^^^^
+
+To configure OMERO to be able to log to syslog, it is necessary to modify the
+file ``OMERO.server/current/etc/logback.xml``. It is possible to do all the
+configuration changes in this file alone, but for ease of config management,
+it is demonstrated here where an additional
+``OMERO.server/current/etc/logback_syslog.xml`` file is used in addition.
+
+The following information is required to configure OMERO to log to syslog.
+
+- The host on which syslog is running: e.g. `localhost`
+- The port number on which syslog is running on that host: e.g. `514`
+- The facility (`RFC 3164 <https://tools.ietf.org/html/rfc3164>`_) that OMERO
+  should be handled as: e.g. `user` or `local6`
+
+.. note::
+
+    The facility is important because it determines how syslog will handle the
+    messages it receives. It is unlikely that OMERO's log output will be desired
+    in a local systems primary message log for example. On Linux this is often
+    ``/var/log/messages``. Remember to configure the syslog configuration
+    to avoid this. This is also where configuration of onward forwarding can
+    be configured (to a service such as `splunk <http://www.splunk.com/>`_).
+    Finally, syslog can be configured to specifically output this facility
+    output to a file such as ``/var/log/omero``.
+
+Create the new file ``OMERO.server/current/etc/logback_syslog.xml``:
+
+::
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <included>
+        <!-- syslog -->
+      <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
+
+        <!-- Exclude debug level logging from ome.services.blitz.repo.ManagedImportRequestI -->
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+          <evaluator> <!-- defaults to type ch.qos.logback.classic.boolex.JaninoEventEvaluator -->
+            <expression>return Level.DEBUG.equals(Level.toLevel(level)) &amp;&amp; logger.equals("ome.services.blitz.repo.ManagedImportRequestI");</expression>
+          </evaluator>
+          <OnMismatch>NEUTRAL</OnMismatch>
+          <OnMatch>DENY</OnMatch>
+        </filter>
+        <!-- Exclude debug level logging from omero.* (except allow omero.cmd.*) -->
+        <filter class="ch.qos.logback.core.filter.EvaluatorFilter">
+          <evaluator> <!-- defaults to type ch.qos.logback.classic.boolex.JaninoEventEvaluator -->
+            <expression>return Level.DEBUG.equals(Level.toLevel(level)) &amp;&amp; logger.startsWith("omero.") &amp;&amp; !logger.startsWith("omero.cmd.");</expression>
+          </evaluator>
+          <OnMismatch>NEUTRAL</OnMismatch>
+          <OnMatch>DENY</OnMatch>
+        </filter>
+
+        <syslogHost>localhost</syslogHost>
+        <facility>local6</facility>
+        <suffixPattern>OMERO [%level] [%thread] %logger %msg</suffixPattern>
+      </appender>
+
+    </included>
+
+This creates an appender that logs to syslog. `syslogHost` is the host on
```

Comment:
I don't know whether all uses of 'syslog' need to be marked up or not. /cc @hflynn